build/zip-examples.js

```diff
@@ -15,34 +15,62 @@ const sh = require('shelljs')
 const pkg = require('../package.json')
 
 const versionShort = pkg.config.version_short
-const folderName = `bootstrap-${pkg.version}-examples`
+const distFolder = `bootstrap-${pkg.version}-examples`
+const rootDocsDir = '_gh_pages'
+const docsDir = `${rootDocsDir}/docs/${versionShort}/`
+
+// these are the files we need in the examples
+const cssFiles = [
+  'bootstrap.min.css',
+  'bootstrap.min.css.map'
+]
+const jsFiles = [
+  'bootstrap.bundle.min.js',
+  'bootstrap.bundle.min.js.map'
+]
+const imgFiles = [
+  'bootstrap-outline.svg',
+  'bootstrap-solid.svg'
+]
 
 sh.config.fatal = true
 
-if (!sh.test('-d', '_gh_pages')) {
-  throw new Error('The "_gh_pages" folder does not exist, did you forget building the docs?')
+if (!sh.test('-d', rootDocsDir)) {
+  throw new Error(`The "${rootDocsDir}" folder does not exist, did you forget building the docs?`)
 }
 
 // switch to the root dir
 sh.cd(path.join(__dirname, '..'))
 
-// remove any previously created folder with the same name
-sh.rm('-rf', folderName)
+// remove any previously created folder/zip with the same name
+sh.rm('-rf', [distFolder, `${distFolder}.zip`])
+
```

Comment:
But the `distFolder` is removed in the end. So initially it would not exist IMO :thinking:

Reply:
True, but I might have it locally while testing.

Reply:
And also if the zip file is updated if it already exists.
When `release-zip-examples` script is executed in real? At the time of release? :thinking:

Reply:
It shouldn't be updated since it's removed beforehand:

```js
sh.rm('-rf', [distFolder, `${distFolder}.zip`])
```

Reply:
Updated if I remove this full line from the code. Not sure if it is needed.

Reply:
Not sure what you mean. I want to remove the folder and the zip file so that we are safe. I definitely don't want the zip file to be updated, which is what we do on main right now. This just backports the main patches

Reply:
Okay, understood :+1: